public/locales/en/translation.json

```diff
@@ -0,0 +1,17 @@
+{
+  "welcome": {
+    "message": "Welcome to React"
+  },
+  "React": {
+    "1": "HEY GUUYS"
+  },
+  "en": {
+    "1": "English"
```

Comment:
these should just be
`"en": "English"` like this

never should be using something like "1" (not useful name)